Formula/nauty.rb

```diff
@@ -13,25 +13,26 @@ class Nauty < Formula
   end
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
```

Comment:
Why do you remove those two settings?

Reply:
Because they're not supported in the current `configure` script.